parser/SqlParser.ypp

```diff
@@ -1332,6 +1369,83 @@ opt_limit_clause:
     }
   }
 
+opt_window_clause:
+  {
+    $$ = nullptr;
+  }
+  | window_declaration_list
+
+window_declaration_list:
+  window_declaration {
+    $$ = new quickstep::PtrList<quickstep::ParseWindow>();
+    $$->push_back($1);
+  }
+  | window_declaration_list window_declaration {
+    $$ = $1;
+    $$->push_back($2);
+  }
+
+window_declaration:
```

Comment:
First, we should sort them whenever possible.
In addition, in thiw PR we should commit code sorted in alphabetic order.
Finally, there are some committed but unsorted code, so we should fix in another separate PR.